ansible/roles/condor_configuration/templates/20-job-user.config.j2

```diff
@@ -0,0 +1,2 @@
+TRUST_UID_DOMAIN = True
+UID_DOMAIN = cyverse.org
```

Comment:
I didn't catch this before, but this should be configurable. I'll fix it in another commit shortly.